libs/wingsdk/test/target-sim/__snapshots__/table.test.ts.snap

```diff
@@ -831,14 +721,7 @@ exports[`tryGet row 2`] = `
   "simulator.json": {
     "resources": [
       {
-        "attrs": {},
-        "path": "root/cloud.TestRunner",
-        "props": {
-          "tests": {},
-        },
-        "type": "@winglang/sdk.std.TestRunner",
-      },
-      {
+        "addr": "c8e507baf046ceb6108a9fe64c08a39ccc39ffe11c",
```

Comment:
These addresses are stable across compiles / builds since they're just the construct node.addr